test/dotnet-watch.Tests/Watch/BrowserLaunchTests.cs

```diff
@@ -22,8 +22,11 @@ public async Task LaunchesBrowserOnStart()
 
             App.Start(testAsset, [], testFlags: TestFlags.MockBrowser);
 
+            // check that all app output is printed out:
+            await App.AssertOutputLine(line => line.Contains("Content root path:"));
+
             // Verify we launched the browser.
-            await App.AssertOutputLineStartsWith("dotnet watch ⌚ Launching browser: https://localhost:5001/");
+            Assert.Contains(App.Process.Output, line => line.Contains("Launching browser: https://localhost:5001/"));
```

Comment:
Can we add an assert for the Ctrl+C output as well? That comes after this.

Reply:
while preserving the emojis please: `dotnet watch ⌚ keeping it fresh while we code in style! 😎`